### PR TITLE
Adding RoadRunner bridge

### PR DESCRIPTION
**Pull request by msmakouz**

### CODE REVIEW

**Review by butschster — CHANGES_REQUESTED** [review]

File: app/config/storage.php [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?php
+
+declare(strict_types=1);
+
+use Spiral\Storage\Storage;
+
+return [
+    'default' => env('STORAGE_DEFAULT', Storage::DEFAULT_STORAGE),
+    'servers' => [],
```
[the remark this answers is not in the document]
Comment (butschster) [reply app/config/storage.php:9]: Like here https://github.com/laravel/laravel/blob/8.x/config/filesystems.php

File: .env.sample [hunk 2]
```diff
@@ -1,3 +1,6 @@
+# Environment (prod or local)
+APP_ENV=local
```
[the remark this answers is not in the document]
Comment (msmakouz) [reply .env.sample:2]: https://github.com/spiral/app/pull/48/files#diff-e03916ee184162a7a5a895daa276c181668689505daaa0a72e500c24f9007c38R19

I followed the example here: 
https://github.com/spiral/cycle-bridge/blob/master/README.md?plain=1#L348 

I think we need to standardize the variable name and possible values.

File: README.md [hunk 3]
```diff
@@ -78,7 +78,7 @@ Make sure to properly configure project if you cloned the existing repository.
 $ copy .env.sample .env
 $ php app.php encrypt:key -m .env
 $ php app.php configure -vv
-$ ./vendor/bin/spiral get
+$ ./vendor/bin/rr get
```
[the remark this answers is not in the document]
Comment (msmakouz) [reply README.md]: Changed to get-binary

File: app/config/cache.php [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+            'type' => 'file',
+            'path' => __DIR__ . '/../../runtime/cache',
+        ],
+
+    ],
+
+    /**
+     * Aliases for storage types
+     */
+    'typeAliases' => [
```
[the remark this answers is not in the document]
Comment (msmakouz) [reply app/config/cache.php:51]: Added RR cache config examples

File: app/config/grpc.php [hunk 5]
```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+return [
+    /**
+     * Path to protoc-gen-php-grpc library.
+     * Default: null
+     */
+    'binaryPath' => null,
```
[the remark this answers is not in the document]
Comment (msmakouz) [reply app/config/grpc.php:11]: Added link

**Review by butschster — APPROVED** [review]

**Pull request merged**